source/flow/tcp/tcp_flow.cpp

```diff
@@ -24,7 +24,7 @@ TcpFlow::TcpFlow(Id a_id, std::shared_ptr<IConnection> a_conn,
       m_last_ack_arrive_time(0),
       m_packet_size(a_packet_size),
       m_ecn_capable(a_ecn_capable),
-      m_packets_in_flight(0),
+      m_inflight(0),
```

Comment:
I think previous naing vresion is suits better:
```suggestion
      m_packets_in_flight(0),
```